play-media-native/features/StackScreenHeader/StackScreenHeader.tsx

```diff
@@ -1,39 +1,110 @@
-import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import { faArrowLeft, faSearch } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
 import { useNavigation } from '@react-navigation/native';
-import { View } from 'react-native';
-import { IconButton } from 'react-native-paper';
+import { useEffect, useMemo, useState } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { IconButton, Text } from 'react-native-paper';
 
-import { Logo } from '../../components/Logo/Logo';
-import { TabHeaderNavigationProp } from '../../interfaces/navigators';
+import { useFilters } from '../../hooks/useFilters/useFilters';
+import { StackNavigationProp } from '../../interfaces/navigators';
 import { theme } from '../../theme/theme';
 
-export const StackScreenHeader = () => {
-  const navigation = useNavigation<TabHeaderNavigationProp>();
+const styles = StyleSheet.create({
+  headerContainer: {
+    paddingTop: theme.spacing.xs,
+    paddingBottom: theme.spacing.xs,
+    paddingLeft: theme.spacing.sm,
+    paddingRight: theme.spacing.sm,
+    backgroundColor: theme.colors.black.darkest,
+  },
+  headerInfo: {
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'space-around',
+  },
+  title: {
+    paddingTop: theme.spacing.xs,
+    paddingHorizontal: theme.spacing.lg,
+    fontSize: theme.fontSize.xl,
+  },
+  subtitle: {
+    fontSize: theme.fontSize.xs,
+    color: theme.colors.gray.DEFAULT,
+  },
+  backBtn: {
+    position: 'absolute',
+    left: 0,
+    top: 0,
```

Comment:
Back button cannot be pressed, maybe it needs a zIndex?

Reply:
Hmm not getting this when navigating away from a create screen, either by clicking the back button or by clicking discard. Might be an iOS bug.

Could you tell me the complete order of steps so that I can try again?

Reply:
I get it when swiping the screen to go back, I cannot go back any other way because the back button is not clickable

Reply:
It might be iOS, I'll check :)

Reply:
Thanks!